src/zmtp_engine.hpp

```diff
@@ -119,9 +119,6 @@ class zmtp_engine_t ZMQ_FINAL : public stream_engine_base_t
     unsigned char _greeting_recv[v3_greeting_size];
     unsigned char _greeting_send[v3_greeting_size];
 
-    //  Size of greeting received so far
-    unsigned int _greeting_bytes_read;
```

Comment:
why was this moved to the base class?

Reply:
Corrected -- an earlier version of the code was checking for greeting_bytes_read > 0 in stream_engine_base.ccp, but that code was found to be superfluous.  See d9634e5b7bbc8168f568797fd0df0b94d1869913